tests/functional/spec/features/calculated_summary.spec.js

```diff
@@ -92,25 +92,18 @@ describe("Feature: Calculated Summary", () => {
       $(CurrencyTotalPlaybackPageWithFourth.fourthNumberAnswerEdit()).click();
```

Comment:
I think it would be good to show functionally, the behaviour of returning to a calculated summary. We should have the same test as we do for section summary. That I am return to the summary and that the anchor exists as expected.

Reply:
Added functional tests 👍